install/bootstrap_livecd_remote_iso.md

```diff
@@ -161,128 +220,120 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
+   The `parted` command may give an error similar to the following:
+   ```text
+   Error: Partition(s) 4 on /dev/sda have been written, but we have been unable to inform the kernel of the change, probably
+   because it/they are in use. As a result, the old partition(s) will remain in use. You should reboot now before making
+   further changes.
+   ```
 
-   **Important:** In an earlier step, the CSM release plus any patches or hotfixes
-   was downloaded to a system using the instructions in [Update CSM Product Stream](../update_product_stream/index.md).
-   Either copy from that system to the PIT node or set the ENDPOINT variable to URL and use `wget`.
+   In that case, the following steps may resolve the problem without needing to reboot. These commands remove
+   volume groups and raid arrays that may be using the disk. **These commands only need to be run if the earlier
+   `parted` command failed.**
 
-   1. Set helper variables
+   ```bash
+   pit# RAIDS=$(grep "${disk}[0-9]" /proc/mdstat | awk '{ print "/dev/"$1 }') ; echo $RAIDS
+   pit# VGS=$(echo $RAIDS | xargs -r pvs --noheadings -o vg_name 2>/dev/null) ; echo $VGS
+   pit# echo $VGS | xargs -r -t -n 1 vgremove -f -v
+   pit# echo $RAIDS | xargs -r -t -n 1 mdadm -S -f -v
+   ```
+
+   After running the above procedure, retry the `parted` command which failed. If it succeeds, resume the install from that point.
+
+1. Mount local disk.
+
+   > **Note:** The FSLabel `PITDATA` is already in `/etc/fstab`, so the path is omitted in the following call to `mount`.
+
+   ```bash
+   pit# mount -vL PITDATA &&
+        mkdir -pv $PITDATA/{admin,configs} $PITDATA/prep/{admin,logs} $PITDATA/data/{k8s,ceph}
+   ```
+
+1. Relocate the typescript to the newly mounted `PITDATA` directory.
+
+   1. Quit the typescript session with the `exit` command.
+
+   1. Copy the typescript file to its new location.
 
       ```bash
-      pit# ENDPOINT=https://arti.dev.cray.com/artifactory/shasta-distribution-stable-local/csm
-      pit# export CSM_RELEASE=csm-x.y.z
-      pit# export SYSTEM_NAME=eniac
+      pit# cp -v ~/csm-install-remoteiso.*.txt $PITDATA/prep/admin
       ```
 
-   1. Save the `CSM_RELEASE` for usage later; all subsequent shell sessions will have this variable set.
+   1. Restart the typescript, appending to the previous file.
+
+      ```bash
+      pit# script -af $(ls -tr $PITDATA/prep/admin/csm-install-remoteiso*.txt | head -n 1)
+      pit# export PS1='\u@\H \D{%Y-%m-%d} \t \w # '
+      ```
+
+1. Download the CSM software release to the PIT node.
 
-      > The `echo` prepends a newline to ensure that the variable assignment occurs on a unique line,
-      > and not at the end of another.
+   1. Set variable to URL of CSM tarball.
 
-      ```bash      
-      pit# echo -e "\nCSM_RELEASE=$CSM_RELEASE" >>/etc/environment
+      ```bash
+      pit# URL=https://arti.dev.cray.com/artifactory/shasta-distribution-stable-local/csm/${CSM_RELEASE}.tar.gz
```

Comment:
Should this URL now be pointing to https://artifactory.algol60.net/artifactory/releases/csm?  arti no longer has the latest releases.   Or should this just be some bogus URL?

Reply:
No idea on that one. I just left the URL that was already being used. I don't know where customers are pulling it down from. Since my PR isn't changing the URL (just relocating it), we can pursue that question with a different ticket, unless you object. If you're cool with that, I can get a new ticket opened to answer that.

Reply:
Okay, we should probably open a separate ticket for this since arti will not work.